tpm2tools/keys.go

```diff
@@ -273,7 +283,18 @@ func (k *Key) Unseal(in *tpmpb.SealedBytes, cOpt CertifyOpt) ([]byte, error) {
 		creationHash := sessionHashAlg.New()
 		creationHash.Write(in.GetCreationData())
 		if _, _, err = tpm2.CertifyCreation(k.rw, "", sealed, tpm2.HandleNull, nil, creationHash.Sum(nil), tpm2.SigScheme{}, ticket); err != nil {
-			return nil, fmt.Errorf("failed to certify creation: %v", err)
+			// Fail to CertifyCreation using a null key handle. It is possible that some firmware cannot take care of the
+			// null key handle.
+			// Try to CertifyCreation again with a dummy AIK key.
+			// Using the AIK is just a workaround, we still ignore the signature and attstation returned by CertifyCreation
+			signer, err := AttestationIdentityKeyECC(k.rw)
+			if err != nil {
+				return nil, fmt.Errorf("fail to create a AIK ECC key: %v", err)
```

Comment:
"failed to create fallback signing key: %v"

Reply:
done